inst/extdata/inconv_factor.csv

```diff
@@ -1,513 +1,569 @@
-SSPscen,techscen,param,value
```

Comment:
Same here. If you have problems with excel in that regard, please use R where you can control the format.